platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -170,7 +167,7 @@ extension PolyOut {
         // actually exist - for now we just ignore them in readDir, but it
         // might be smarter to clean fileStore automatically at some point.
         let storedFiles = fileStore.keys.filter { key in
```

Comment:
@GhenadiePusca this is the place where the use of fileStore does not make much sense to me. Why give the contents of the folders in the case of `url != ""` and give the names of the archives in the other case? 

It is strange